NLogInjector.UnitTests/NLogInjector.UnitTests.csproj

```diff
@@ -31,23 +34,36 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Autofac, Version=4.4.0.0, Culture=neutral, PublicKeyToken=17863af14b0044da, processorArchitecture=MSIL">
-      <HintPath>..\packages\Autofac.4.4.0\lib\net45\Autofac.dll</HintPath>
+    <Reference Include="Autofac, Version=4.9.2.0, Culture=neutral, PublicKeyToken=17863af14b0044da, processorArchitecture=MSIL">
```

Comment:
Is this the right version of Autofac for .net standard? They have an `Autofac.Extensions.DependencyInjection`, I will have to pull the code down and check it out, just wondering if you looked into that option?

Reply:
This version has a netstandard dll in it.
Seems `Autofac.Extensions.DependencyInjection` just implements the interfaces of `Microsoft.Extensions.DependencyInjection.Abstractions`